### DISCUSSION TITLE
Visual Studio for Mac retiring and .NET 8 MAUI application

### DISCUSSION THREAD

**Discussion by agingo** [dotnet/maui#19022] [General]
We've been using Visual Studio for Mac (VSM) to debug our Catalyst applications. Now that VSM is retiring and no longer receiving updates (for .NET for example) is there going to be a way to remotely debug from Visual Studio on Windows to a Mac?

Or do we have to convert our MAUI applications to VS Code when MAUI support is fully released?

**Comment by davidortinau** [comment]
For Mac Catalyst apps you should use VS Mac and/or VS Code. The same project works in both and doesn't require any conversion. Please file feedback on any issues you have using those tools.

**Reply by agingo** [reply]
@davidortinau Do you know when the MAUI extension for VSCode (OSX) will be out of preview mode?
